internal/render/common/configmap.go

```diff
@@ -113,7 +115,24 @@ func generateSlurmConfig(cluster *values.SlurmCluster) renderutils.ConfigFile {
 	res.AddComment("COMPUTE NODES")
 	res.AddComment("We're using the \"dynamic nodes\" feature: https://slurm.schedmd.com/dynamic_nodes.html")
 	res.AddProperty("MaxNodeCount", "512")
-	res.AddProperty("PartitionName", "main Nodes=ALL Default=YES MaxTime=INFINITE State=UP OverSubscribe=YES")
+	res.AddComment("Partition Configuration")
+	res.AddProperty("JobRequeue", 1)
+	res.AddProperty("PreemptMode", "REQUEUE")
+	res.AddProperty("PreemptType", "preempt/partition_prio")
+	switch cluster.PartitionConfiguration.ConfigType {
+	case "default":
+		res.AddProperty("PartitionName", "main Nodes=ALL Default=YES MaxTime=INFINITE State=UP OverSubscribe=YES")
+	case "custom":
+		scanner := bufio.NewScanner(strings.NewReader(cluster.PartitionConfiguration.RawConfig))
+		for scanner.Scan() {
+			line := strings.TrimSpace(scanner.Text())
+
+			if strings.HasPrefix(line, "PartitionName") {
```

Comment:
Why there is no validation anymore?

I think the logic should be such:
- Lines that start with "PartitionName" are partition configuration that is inserted into the slurm.conf
- Empty lines, lines of blank chars, and lines starting with "#" are ignored.

We should fail if a user tries to configure "PreemptMode" or anything else except "PartionName=....". Now such settings are just silently ignored, and I don't think it's a good idea.

Reply:
You wrote exactly that I did in code) 

Why we should fail? We wrote example that config lines should have PartionName in the start of the line. And only them we get from config. How you want validate another string?

Reply:
OK, but what if the user writes "PartirionName=..." (with a mistake) -- this line will be ignored.

Reply:
So, what will be if user write #PartirionName=..? 
Next question: where is better place for fail? In operator (here you need read logs of operator in order to understand what happen and where is an error) or in slurm (read that in sinfo command or in slurm log)?